firmware/config/boards/kinetis/config/controllers/algo/engine_configuration_generated_structures.h

```diff
@@ -2052,7 +2052,11 @@ struct engine_configuration_s {
 	/**
 	 * offset 1772
 	 */
-	uint8_t unusedFormerWarmupAfrPid[4];
+	uint8_t unusedFormerWarmupAfrPid[3];
```

Comment:
please put the new field at the BEGINNING of the unused block

Reply:
new field is already in master. BEGINNING is better but probably not a huge difference?